src/cloud/docker/docker-extend.md

```diff
@@ -46,4 +46,22 @@ If you want to update the container configuration and test iteratively, run the
 docker-compose up -d --force-recreate --build
 ```
 
+## Building the Magento Cloud Docker PHP Images
```

Comment:
In the link reference in the Docker service containers topic, the title is  "Rebuilding the ...."